feature-group-definitions/custom-highlight.yml

```diff
@@ -0,0 +1,3 @@
+name: Custom highlights
```

Comment:
Commenting on the first line for lack of a better place. I think the shortest identifier we can get away with here is just `highlight` (`highlight.yml`) since that matches both the `Highlight` constructor and the `::highlight()` pseudo-element.

Reply:
Very surprising to me, because `highlight` is kind of a generic name, but it indeed looks like nothing else on the web platform uses that name yet. So that might work.

Reply:
I take it you're not a fan of some of our shortest identifiers like `search`, `set`, or `where`. Perhaps it was a mistake to try to make identifiers as short as possible and use a bit of syntax where possible. But I think `highlight` fits with the current approach, so let's go with that for now.

Reply:
@captainbrosset are you OK with renaming this to `highlight.yml`? I think that will stand the test of time even if we change the name and description.